{"uid":"bf8892f7226d7fa8","name":"View account settings succesfully","fullName":"View account settings succesfully","historyId":"View account settings succesfully","time":{"start":1677727428940,"stop":1677727428946,"duration":6},"status":"broken","statusMessage":"An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:4444/session/868ef3810cb8c89d0e54880adc75a84e/screenshot. The exception message was: Connection refused (localhost:4444) -> Connection refused (localhost:4444) -> Connection refused","statusTrace":"OpenQA.Selenium.WebDriverException: An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:4444/session/868ef3810cb8c89d0e54880adc75a84e/screenshot. The exception message was: Connection refused (localhost:4444)\n ---> System.Net.Http.HttpRequestException: Connection refused (localhost:4444)\n ---> System.Net.Sockets.SocketException (111): Connection refused\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)\n   at System.Net.Sockets.Socket.<ConnectAsync>g__WaitForConnectWithCancellation|277_0(AwaitableSocketAsyncEventArgs saea, ValueTask connectTask, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.ConnectToTcpHostAsync(String host, Int32 port, HttpRequestMessage initialRequest, Boolean async, CancellationToken cancellationToken)\n   --- End of inner exception stack trace ---\n   at System.Net.Http.HttpConnectionPool.ConnectToTcpHostAsync(String host, Int32 port, HttpRequestMessage initialRequest, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.CreateHttp11ConnectionAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.AddHttp11ConnectionAsync(HttpRequestMessage request)\n   at System.Threading.Tasks.TaskCompletionSourceWithCancellation`1.WaitWithCancellationAsync(CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.GetHttp11ConnectionAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.SendWithVersionDetectionAndRetryAsync(HttpRequestMessage request, Boolean async, Boolean doRequestAuth, CancellationToken cancellationToken)\n   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpClient.<SendAsync>g__Core|83_0(HttpRequestMessage request, HttpCompletionOption completionOption, CancellationTokenSource cts, Boolean disposeCts, CancellationTokenSource pendingRequestsCts, CancellationToken originalCancellationToken)\n   at OpenQA.Selenium.Remote.HttpCommandExecutor.MakeHttpRequest(HttpRequestInfo requestInfo)\n   at OpenQA.Selenium.Remote.HttpCommandExecutor.Execute(Command commandToExecute)\n   --- End of inner exception stack trace ---\n   at OpenQA.Selenium.Remote.HttpCommandExecutor.Execute(Command commandToExecute)\n   at OpenQA.Selenium.WebDriver.Execute(String driverCommandToExecute, Dictionary`2 parameters)\n   at OpenQA.Selenium.WebDriver.GetScreenshot()\n   at SeleniumTest.Tests.Hooks.Hooks.CaptureScreenshot() in /home/sch/actions-runner/_work/GUI-Framework/GUI-Framework/3.Tests/UI/Hooks/Hooks.cs:line 176\n   at TechTalk.SpecFlow.Bindings.BindingInvoker.InvokeBinding(IBinding binding, IContextManager contextManager, Object[] arguments, ITestTracer testTracer, TimeSpan& duration)\n   at Allure.SpecFlowPlugin.AllureBindingInvoker.InvokeBinding(IBinding binding, IContextManager contextManager, Object[] arguments, ITestTracer testTracer, TimeSpan& duration)","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"BeforeFeature [10000]","time":{"start":1677727428929,"stop":1677727428939,"duration":10},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"CreateProject [1]","time":{"start":1677727428940,"stop":1677727428940,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"CreateItems [2]","time":{"start":1677727428940,"stop":1677727428940,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"BeforeScenario [10000]","time":{"start":1677727428940,"stop":1677727428940,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"testStage":{"status":"broken","statusMessage":"An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:4444/session/868ef3810cb8c89d0e54880adc75a84e/screenshot. The exception message was: Connection refused (localhost:4444) -> Connection refused (localhost:4444) -> Connection refused","statusTrace":"OpenQA.Selenium.WebDriverException: An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:4444/session/868ef3810cb8c89d0e54880adc75a84e/screenshot. The exception message was: Connection refused (localhost:4444)\n ---> System.Net.Http.HttpRequestException: Connection refused (localhost:4444)\n ---> System.Net.Sockets.SocketException (111): Connection refused\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)\n   at System.Net.Sockets.Socket.<ConnectAsync>g__WaitForConnectWithCancellation|277_0(AwaitableSocketAsyncEventArgs saea, ValueTask connectTask, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.ConnectToTcpHostAsync(String host, Int32 port, HttpRequestMessage initialRequest, Boolean async, CancellationToken cancellationToken)\n   --- End of inner exception stack trace ---\n   at System.Net.Http.HttpConnectionPool.ConnectToTcpHostAsync(String host, Int32 port, HttpRequestMessage initialRequest, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.CreateHttp11ConnectionAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.AddHttp11ConnectionAsync(HttpRequestMessage request)\n   at System.Threading.Tasks.TaskCompletionSourceWithCancellation`1.WaitWithCancellationAsync(CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.GetHttp11ConnectionAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.SendWithVersionDetectionAndRetryAsync(HttpRequestMessage request, Boolean async, Boolean doRequestAuth, CancellationToken cancellationToken)\n   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpClient.<SendAsync>g__Core|83_0(HttpRequestMessage request, HttpCompletionOption completionOption, CancellationTokenSource cts, Boolean disposeCts, CancellationTokenSource pendingRequestsCts, CancellationToken originalCancellationToken)\n   at OpenQA.Selenium.Remote.HttpCommandExecutor.MakeHttpRequest(HttpRequestInfo requestInfo)\n   at OpenQA.Selenium.Remote.HttpCommandExecutor.Execute(Command commandToExecute)\n   --- End of inner exception stack trace ---\n   at OpenQA.Selenium.Remote.HttpCommandExecutor.Execute(Command commandToExecute)\n   at OpenQA.Selenium.WebDriver.Execute(String driverCommandToExecute, Dictionary`2 parameters)\n   at OpenQA.Selenium.WebDriver.GetScreenshot()\n   at SeleniumTest.Tests.Hooks.Hooks.CaptureScreenshot() in /home/sch/actions-runner/_work/GUI-Framework/GUI-Framework/3.Tests/UI/Hooks/Hooks.cs:line 176\n   at TechTalk.SpecFlow.Bindings.BindingInvoker.InvokeBinding(IBinding binding, IContextManager contextManager, Object[] arguments, ITestTracer testTracer, TimeSpan& duration)\n   at Allure.SpecFlowPlugin.AllureBindingInvoker.InvokeBinding(IBinding binding, IContextManager contextManager, Object[] arguments, ITestTracer testTracer, TimeSpan& duration)","steps":[{"name":"Given  the user is logged in","time":{"start":1677727428941,"stop":1677727428942,"duration":1},"status":"failed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"attachments":[],"parameters":[],"stepsCount":1,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[{"name":"CaptureScreenshot [10000]","time":{"start":1677727428946,"stop":1677727428947,"duration":1},"status":"broken","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"AfterFeature [10000]","time":{"start":1677727428950,"stop":1677727428950,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"labels":[{"name":"thread","value":"NonParallelWorker"},{"name":"host","value":"sch"},{"name":"feature","value":"Settings account view"},{"name":"tag","value":"Regression"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":8,"unknown":0,"total":9},"items":[{"uid":"463b43c9880a5e26","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/732//#testresult/463b43c9880a5e26","status":"passed","time":{"start":1677726566792,"stop":1677726573235,"duration":6443}},{"uid":"84e85fa492b79095","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/730//#testresult/84e85fa492b79095","status":"passed","time":{"start":1677724893344,"stop":1677724899805,"duration":6461}},{"uid":"b71716fe7a09ce87","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/14//#testresult/b71716fe7a09ce87","status":"passed","time":{"start":1677718750200,"stop":1677718756901,"duration":6701}},{"uid":"817d5150a94aa0fc","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/729//#testresult/817d5150a94aa0fc","status":"passed","time":{"start":1677714527063,"stop":1677714533895,"duration":6832}},{"uid":"d1a104cc975c77c1","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/728//#testresult/d1a104cc975c77c1","status":"passed","time":{"start":1677713154091,"stop":1677713161429,"duration":7338}},{"uid":"5080ba6baabe17fb","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/727//#testresult/5080ba6baabe17fb","status":"passed","time":{"start":1677710617776,"stop":1677710624284,"duration":6508}},{"uid":"2e0b5c5b13dbedeb","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/726//#testresult/2e0b5c5b13dbedeb","status":"passed","time":{"start":1677709855515,"stop":1677709861941,"duration":6426}},{"uid":"d59c1041e41e243d","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/725//#testresult/d59c1041e41e243d","status":"passed","time":{"start":1677709307508,"stop":1677709314103,"duration":6595}}]},"tags":["Regression"]},"source":"bf8892f7226d7fa8.json","parameterValues":[]}